sdk/couchbase-core/src/analyticscomponent.rs

```diff
@@ -82,10 +82,7 @@ impl<C: Client> AnalyticsComponent<C> {
         ))
     }
 
-    pub async fn query<'a>(
-        &self,
-        opts: &AnalyticsOptions<'a>,
-    ) -> error::Result<AnalyticsResultStream> {
+    pub async fn query<'a>(&self, opts: AnalyticsOptions) -> error::Result<AnalyticsResultStream> {
```

Comment:
Does this function still need the lifetime specifier?

Reply:
No, I'm surprised clippy didn't shout about it tbh.